frontend/src/components/user/searchdetails/SearchUserDetailsView.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import Avatar from '@mui/material/Avatar'
+import classes from './searchuserdetails.module.scss'
+import FollowButtonController from '../../button/followbutton/FollowButtonController'
+
+/**
+ * The SearchUserDetailsView component is used to display the any results for users while performing a search.
+ *
+ * Props:
+ * @username : the username of the appropriate user.
+ * @handle : the handle (@) of the appropriate user.
+ * @bio : the biography of the appropriate user.
+ * @profilePic : the link to the profile image of the appropriate user.
+ *
+ * The FollowButton function has two props isFollowing and OnClick which are can be change to appropriate values when needed.
```

Comment:
Might need to say "... and OnClick which can be changed to appropriate values..."

Reply:
My brain at 1 am doesn't know English haha. Thank you for pointing it out :)